cypress/support/step-definitions/select-steps.js

```diff
@@ -25,48 +22,30 @@ import {
 import { dataComponentButtonByText } from "../../locators/pages";
 import { loader } from "../../locators/loader";
 
-When("I focus select input", () => {
-  simpleSelectID().focus();
-});
-
-When("I focus default Select input", () => {
-  simpleSelect().focus();
-});
-
-When("I focus openOnFocus Select input", () => {
-  openOnFocusID().focus();
-});
-
-When("I click openOnFocus Select input", () => {
-  openOnFocusID().click({ force: true });
-});
-
-Then("{string} Select list is opened", (name) => {
-  selectDataComponent(name).should("have.attr", "aria-expanded", "true");
+Then("{string} list is open", () => {
+  selectInput().should("have.attr", "aria-expanded", "true");
   selectList().should("be.visible");
 });
 
-Then("{string} Select list is closed", (name) => {
-  selectDataComponent(name).should("have.attr", "aria-expanded", "false");
+Then("{string} list is closed", () => {
```

Comment:
`{string}` is useless here - because we are not providing it down to the TC

Reply:
it's being used in three separate components, and we're providing their names here